pkg/filters/cgroup-bpf/bpf.go

```diff
@@ -0,0 +1,118 @@
+package bpf
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"time"
+	"unsafe"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/link"
+	"golang.org/x/sys/unix"
+)
+
+type bpf struct {
+	prog *ebpf.Program
+}
+
+const (
+	rootCgroup     = "/sys/fs/cgroup/unified"
```

Comment:
Note to myself: we have done the previous benchmarks with a Flatcar version without the cgroup hybrid mode, so cgroup2 is not mounted. We will need the hybrid mode for this to work.